src/Traits/HasRoles.php

```diff
@@ -277,4 +277,27 @@ protected function getStoredRole($role)
 
         return $role;
     }
+
+    /**
+     * Return all the permissions the user have via his roles.
+     *
+     * @return Collection
+     */
+    protected function getPermissionsViaRolesAttribute()
```

Comment:
@freekmurze @mathieutu in master branch still this is not update. It's just a method not an accessors `getPermissionsViaRoles` . + `getAllPermissions`